### PR TITLE
Add Stakin contributions

### PR DESCRIPTION
**Pull request by geldmon**

### CODE REVIEW

**Review by jcstein — CHANGES_REQUESTED** [review]
hi @geldmon please resolve conflicts

**Comment by geldmon** [comment]
:white_check_mark: @jcstein

File: README.md [hunk 1]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
 
 ## Node operator contributions
 
+######
```
Comment (jcstein) [inline README.md:108]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+### Contributions by [Stakin](https://stakin.com)
+######
```
Comment (jcstein) [inline README.md:110]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+
+### Mainnet:
```
Comment (jcstein) [inline README.md:112]:
```suggestion
#### Mainnet
```

File: README.md [hunk 4]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+
+##### Relayers: We support Celestia by providing relayers channels, this is our relayers wallet for Celestia
```
Comment (jcstein) [inline README.md:114]:
```suggestion
* Relayers: We support Celestia by providing relayers channels, this is our relayers wallet for Celestia 
```

File: README.md [hunk 5]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+### Contributions by [Stakin](https://stakin.com)
+######
+
+### Mainnet:
+
+##### Relayers: We support Celestia by providing relayers channels, this is our relayers wallet for Celestia 
+```
+celestia1q6cc9u0x5r3fkjcex0rgxee5qlu86w8rxq438l
+```
+#### RPC: https://celestia.rpc.stakin-nodes.com
```
Comment (jcstein) [inline README.md:118]:
```suggestion
* RPC: https://celestia.rpc.stakin-nodes.com
```

File: README.md [hunk 6]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+#### gRPC: celestia.grpc.stakin-nodes.com:443
```
Comment (jcstein) [inline README.md:119]:
```suggestion
* gRPC: celestia.grpc.stakin-nodes.com:443
```

File: README.md [hunk 7]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+#### API: https://celestia.rest.stakin-nodes.com
```
Comment (jcstein) [inline README.md:120]:
```suggestion
* API: https://celestia.rest.stakin-nodes.com
```

File: README.md [hunk 8]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+#### Dashboard and staking interface:
```
Comment (jcstein) [inline README.md:121]:
```suggestion
* Dashboard and staking interface: 
```

File: README.md [hunk 9]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+```
+celestia1q6cc9u0x5r3fkjcex0rgxee5qlu86w8rxq438l
+```
+#### RPC: https://celestia.rpc.stakin-nodes.com
+#### gRPC: celestia.grpc.stakin-nodes.com:443
+#### API: https://celestia.rest.stakin-nodes.com
+#### Dashboard and staking interface: 
+- Stakers can monitor their staked TIA and staking rewards on the staking, a free-to-use validator-agnostic tool that anybody can use. [https://dashboard.stakin.com](https://dashboard.stakin.com)
+- The dashboard also features institutional-grade staking features via Fireblocks raw signing
+##### Blog/Articles: [https://stakin.com/blog/tag/celestia](https://stakin.com/blog/tag/celestia)
```
Comment (jcstein) [inline README.md:124]:
```suggestion
* Blog/Articles: [https://stakin.com/blog/tag/celestia](https://stakin.com/blog/tag/celestia)
```

File: README.md [hunk 10]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+
+
+### Testnet:
```
Comment (jcstein) [inline README.md:127]:
```suggestion
#### Testnet:
```

File: README.md [hunk 11]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+
+#### Validator: [celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha](https://testnet.celestia.explorers.guru/validator/celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha)
```
Comment (jcstein) [inline README.md:129]:
```suggestion
* Validator: [celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha](https://testnet.celestia.explorers.guru/validator/celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha)
```

File: README.md [hunk 12]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+##### Bridge:
```
Comment (jcstein) [inline README.md:130]:
```suggestion
* Bridge: 
```

File: README.md [hunk 13]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+- The dashboard also features institutional-grade staking features via Fireblocks raw signing
+##### Blog/Articles: [https://stakin.com/blog/tag/celestia](https://stakin.com/blog/tag/celestia)
+
+
+### Testnet:
+
+#### Validator: [celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha](https://testnet.celestia.explorers.guru/validator/celestiavaloper1wf5az63pmykhnevyjcja68wdgs7h8y4w565lha)
+##### Bridge: 
+```12D3KooWH4wjJvFMwRsxpF78X4BBVpXswNFTM3QbAcashftUFf9u```
+
```
Comment (jcstein) [inline README.md:132]:
```suggestion
```

File: README.md [hunk 14]
```diff
@@ -105,6 +105,33 @@ An Awesome List of Celestia Resources
...
+
```
Comment (jcstein) [inline README.md:133]:
```suggestion
```

**Comment by jcstein** [comment]
please apply changes so it fits current format. thanks!

**Comment by geldmon** [comment]
✅  @jcstein

**Pull request merged**